client/src/views/TeamListView/TeamListView.jsx

```diff
@@ -1,3 +1,183 @@
-const TeamListView = () => {}
+import axios from 'axios'
+import { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
 
-export default TeamListView
+import Button from '../../components/Button/Button'
+import Card from '../../components/Card/Card'
+import { useAuth } from '../../state/AuthProvider/AuthProvider'
+
+const ProjectView = () => {
```

Comment:
if this is the project view, we should rename the file to `ProjectView.jsx`. Team list view is a different screen.

Reply:
With the current logic TeamListView is not used, but may be useful later so I kept it in.